storybook/storybook-docs/src/interactive-elements.stories.mdx

```diff
@@ -0,0 +1,66 @@
+import { Meta } from "@storybook/blocks";
+
+<Meta title="Docs/Designrichtlijnen/Interactieve elementen" />
+
+# Interactieve elementen
+
+Interactieve elementen zijn onder andere links, knoppen en formuliervelden. Interactieve elementen moeten duidelijk herkenbaar zijn en makkelijk te onderscheiden van niet-interactieve elementen. Je moet ze kunnen gebruiken op alle gangbare manieren, zoals met een muis, toetsenbord en touchscreen.
+
+## Tekst
+
+Interactieve elementen moeten duidelijk zijn, gebruikers moeten kunnen voorspellen wat er gebeurt als je een element activeert. Een interactief element moet ook duidelijk zijn zonder dat je de zinnen er omheen gelezen hebt. Gebruik geen onduidelijke tekst, zoals "hier" of "lees meer".
```

Comment:
We can write a bit more active and terse. Separate pieces of advice can use an empty line between them. And I propose using proper curly quotes.

```suggestion
Gebruikers moeten kunnen voorspellen wat er gebeurt als je een interactief element activeert. Ook zonder de zinnen er omheen te lezen.

Gebruik geen algemene fragmenten voor links, zoals “klik hier” of “lees meer”. Dit is vervelend voor gebruikers van schermlezers die tussen links navigeren. Ook is het niet optimaal voor zoekmachines.
```